### PR TITLE
resource/alicloud_bastionhost_instance: check "enable_public_access" value before update

### PR DESCRIPTION
**Pull request by shanye997**

### CODE REVIEW

File: alicloud/resource_alicloud_bastionhost_instance.go [hunk 1]
```diff
@@ -507,7 +509,7 @@ func resourceAlicloudBastionhostInstanceUpdate(d *schema.ResourceData, meta inte
 		d.SetPartial("security_group_ids")
 	}
 
-	if d.HasChange("enable_public_access") {
+	if d.HasChange("enable_public_access") || (d.IsNewResource() && !d.Get("enable_public_access").(bool)) {
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_bastionhost_instance.go:512]: 这个换成在 Create 中set 系统默认值的逻辑吧

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Review by xiaozhu36 — APPROVED** [review]

**Review by xiaozhu36 — APPROVED** [review]

**Review by xiaozhu36 — APPROVED** [review]

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**